addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -2532,127 +2532,6 @@ export class OdooEditor extends EventTarget {
             this.editable.classList.add('o_row_resize');
         }
     }
-    /**
-     * Resizes a table in the given direction, by "pulling" the border between
-     * the given targets (ordered left to right or top to bottom).
-     *
-     * @private
-     * @param {MouseEvent} ev
-     */
-    _resizeTable(ev, direction, target1, target2) {
-        ev.preventDefault();
-        const position = target1 ? (target2 ? 'middle' : 'last') : 'first';
-        let [item, neighbor] = [target1 || target2, target2];
-        const table = closestElement(item, 'table');
-        const [sizeProp, positionProp, clientPositionProp] = direction === 'col' ? ['width', 'x', 'clientX'] : ['height', 'y', 'clientY'];
-
-        // Preserve current sizes.
-        const tableRect = table.getBoundingClientRect();
-        table.style[sizeProp] = tableRect[sizeProp] + 'px';
-        const unsizedItemsSelector = `${direction === 'col' ? 'td' : 'tr'}:not([style*=${sizeProp}])`;
-        for (const unsizedItem of table.querySelectorAll(unsizedItemsSelector)) {
-            unsizedItem.style[sizeProp] = unsizedItem.getBoundingClientRect()[sizeProp] + 'px';
-        }
-
-        // TD widths should only be applied in the first row. Change targets and
-        // clean the rest.
-        if (direction === 'col') {
-            let hostCell = closestElement(table, 'td');
-            const hostCells = [];
-            while (hostCell) {
-                hostCells.push(hostCell);
-                hostCell = closestElement(hostCell.parentElement, 'td');
-            }
```

Comment:
I don't understand the case that those lines should cover. @Zynton do you remember the reason?

Reply:
I think that's to avoid having conflicting column widths declared in different cells of the same column.
Like:
```html
<tr>
    <td style="width: 100px;">...</td>
</tr>
<tr>
    <td style="width: 200px;">...</td>
</tr>
```